floodviz/static/js/hydrograph.js

```diff
@@ -182,7 +180,8 @@
 		 * Initialize the Hydrograph
 		 */
 		self.init = function (linked_interactions) {
```

Comment:
Move the comments describing the functions from the hydromodule comments at the top to here.